### ISSUE TITLE
ensureIndex with options throws exception

### ISSUE THREAD

**Issue by btknorr** [mafintosh/mongojs#202]
```collection.ensureIndex({ property: 1 }, {unique: true});```

```stack: 
   [ 'TypeError: undefined is not a function',
     '    at /Users/brianknorr/Dev/leisurelink/pmc-api/node_modules/mongojs/lib/collection.js:166:7',
     '    at /Users/brianknorr/Dev/leisurelink/pmc-api/node_modules/mongojs/node_modules/mongodb-core/lib/topologies/server.js:795:13',
     '    at Callbacks.emit (/Users/brianknorr/Dev/leisurelink/pmc-api/node_modules/mongojs/node_modules/mongodb-core/lib/topologies/server.js:94:3)',
     '    at null.messageHandler (/Users/brianknorr/Dev/leisurelink/pmc-api/node_modules/mongojs/node_modules/mongodb-core/lib/topologies/server.js:235:23)',
     '    at Socket.<anonymous> (/Users/brianknorr/Dev/leisurelink/pmc-api/node_modules/mongojs/node_modules/mongodb-core/lib/connection/connection.js:259:22)',
     '    at Socket.emit (events.js:107:17)',
     '    at readableAddChunk (_stream_readable.js:163:16)',
     '    at Socket.Readable.push (_stream_readable.js:126:10)',
     '    at TCP.onread (net.js:538:20)' ] }```

**Comment by saintedlama** [comment]
Pull request #203 fixes the problem. You can fix this in your code by passing a noop callback to ensureIndex

```
var noop = function() {};
db.somecollection.ensureIndex({ field: 1 }, { unique: true }, noop);
```

**Comment by saintedlama** [comment]
Fixed by #203
